### PR TITLE
Fixed errcheck issues in server/channels/app/plugin_install.go

### PR DESCRIPTION
**Pull request by Aryakoste**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/29054
Jira https://mattermost.atlassian.net/browse/MM-61456
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: server/i18n/en.json [hunk 1]
```diff
@@ -5986,6 +5986,10 @@
     "id": "app.plugin.disabled.app_error",
     "translation": "Plugins have been disabled. Please check your logs for details."
   },
+  {
+    "id": "app.plugin.seek.app_error",
+    "translation": "Unable to reset the read position to the start of the plugin bundle."
+  },
```
Comment (hanzei) [inline server/i18n/en.json:5992]: The order of the translations is not correct. You need to run `make i18n-extract` and commit the changes.

**Review by hanzei — CHANGES_REQUESTED** [review]

**Comment by Aryakoste** [comment]
@hanzei Changes done.

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Comment by hanzei** [comment]
@Aryakoste Can you please merge `master` into your branch? Thanks!

**Comment by Aryakoste** [comment]
@hanzei Done

**Comment by hanzei** [comment]
Thanks @Aryakoste :tada:

**Pull request merged**